src/Models/TaskItem.ts

```diff
@@ -0,0 +1,11 @@
+export default class TaskItem{
```

Comment:
You don't need this class. Go with more functional approach. try to replace this with `types` for type safety

Reply:
I have fixed using "type TaskItem" instead of classes.